### PR TITLE
Added CreateGameRequest

### PR DESCRIPTION
**Pull request by rdelfin**
@franspaco @vpineda1996 
Contains PR #9 and #10

### CODE REVIEW

**Comment by rdelfin** [comment]
Please note this includes changes from my other two pull requests

File: src/main/java/com/bunchiestudios/cahserver/requests/CreateGameRequest.java [hunk 1]
```diff
@@ -3,21 +3,83 @@
...
- * Created by rdelfin on 8/27/16.
+ * This is the request made by the user when they want to create a new game. Provided a game name, an optional password,
+ * and some authentication, the appropriate data structures should be set up.
  */
 public class CreateGameRequest extends ServerRequest {
 
-    public CreateGameRequest(DataManager mgr) {
-        super(mgr);
+    public CreateGameRequest(DataManager mgr) throws ProcessingException {
+        super(mgr, "resource:json/schema/request/CreateGame.json", "resource:json/schema/response/CreateGame.json");
```
Comment (vpnda) [inline src/main/java/com/bunchiestudios/cahserver/requests/CreateGameRequest.java:25]: wouldn't it be easier if you move all of this hard coded crap to a constants file?

**Review by vpnda — APPROVED** [review]

File: src/main/java/com/bunchiestudios/cahserver/requests/CreateGameRequest.java (hunk 1 above)
Comment (rdelfin) [reply src/main/java/com/bunchiestudios/cahserver/requests/CreateGameRequest.java:25]: That would, of course, make too much sense.

Hahahaha. I'll create a file for that (config yaml file or something eventually?)

Comment (rdelfin) [reply src/main/java/com/bunchiestudios/cahserver/requests/CreateGameRequest.java:25]: This'll be part of _ANOTHER_ PR

**Pull request merged**